mongodbatlas/resource_mongodbatlas_backup_compliance_policy.go

```diff
@@ -0,0 +1,341 @@
+package mongodbatlas
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/mwielbut/pointy"
+	"github.com/spf13/cast"
+	matlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const (
+	errorBackupPolicyUpdate  = "error updating a Backup Compliance Policy: %s: %s"
+	errorBackupPolicyRead    = "error getting a Backup Compliance Policy for the project(%s): %s"
+	errorBackupPolicySetting = "error setting `%s` for Backup Compliance Policy : %s: %s"
+)
+
+func resourceMongoDBAtlasBackupCompliancePolicy() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceMongoDBAtlasBackupCompliancePolicyCreate,
+		UpdateContext: resourceMongoDBAtlasBackupCompliancePolicyUpdate,
+		ReadContext:   resourceMongoDBAtlasBackupCompliancePolicyRead,
+		DeleteContext: resourceMongoDBAtlasBackupCompliancePolicyDelete,
+		Importer: &schema.ResourceImporter{
+			StateContext: resourceMongoDBAtlasBackupCompliancePolicyImportState,
+		},
+		Schema: map[string]*schema.Schema{
+			"project_id": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"authorized_email": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"copy_protection_enabled": {
+				Type:     schema.TypeBool,
+				Required: true,
+			},
+			"encryption_at_rest_enabled": {
+				Type:     schema.TypeBool,
+				Required: true,
+			},
+			"restore_window_days": {
+				Type:     schema.TypeInt,
+				Optional: true,
+				Computed: true,
+			},
+			"on_demand_policy_item": {
+				Type:     schema.TypeList,
+				MaxItems: 1,
+				Required: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"id": {
+							Type:     schema.TypeString,
+							Computed: true,
+						},
+						"frequency_interval": {
+							Type:     schema.TypeInt,
+							Required: true,
+						},
+						"frequency_type": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"retention_unit": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"retention_value": {
+							Type:     schema.TypeInt,
+							Required: true,
+						},
+					},
+				},
+			},
+			"pit_enabled": {
+				Type:     schema.TypeBool,
+				Required: true,
+			},
+			"state": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"updated_date": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"updated_user": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"scheduled_policy_items": {
```

Comment:
Is there any reason we can't do this similar to schedules? https://github.com/mongodb/terraform-provider-mongodbatlas/blob/master/mongodbatlas/resource_mongodbatlas_cloud_backup_schedule.go#L119